GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift

```diff
@@ -131,6 +133,23 @@ private extension SecureConversations.WelcomeViewModel {
 
         return !filteredQueues.isEmpty
     }
+
+    func fetchSiteConfigurations() {
+        environment.fetchSiteConfigurations { [weak self, alertConfiguration = environment.alertConfiguration] result in
+            switch result {
+            case let .success(site):
+                self?.isAttachmentsAvailable = site.allowedFileSenders.visitor
```

Comment:
From the name of the method, one wouldn't be able to realize that this is also setting the `isAttachmentAvailable` property, just that the configuration is fetched.

Reply:
Fixed 50f39e37a0bc83f9de1c7af639d02464373709c2.